src/extensionFunctions.ts

```diff
@@ -183,6 +192,19 @@ export namespace extensionFunctions {
 		sendTextToOutputChannel(`Sent terminal ${name} the text ${text}`);
 	}
 
+	// send a message to a named terminal
+	export async function sendTerminalCtrlC(name:string) {
+		const terminal : vscode.Terminal | undefined = findTerminal(name);
+		if (!terminal) {
+			const newterminal = vscode.window.createTerminal(name);
```

Comment:
if I understand well, in case, the terminal doesn't exists, we are creating it and then send the Ctrl+C?
I don't understand the use case. If the terminal doesn't exists, shouldn't we better warned user that the terminal cannot be found?

Reply:
That is a good call. I'll update